lib/sarif/bundle_audit_sarif.rb

```diff
@@ -0,0 +1,31 @@
+module Sarif
+  class BundleAuditSarif < BaseSarif
+    BUNDLEAUDIT_URI = 'https://github.com/securego/gosec'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @logs = @scan_report.to_h[:info][:vulnerabilities]
+      @uri = BUNDLEAUDIT_URI
+      @urls = Set.new
+    end
+
+    def parse_issue(issue)
+      return nil if @urls.include?(issue[:url])
+
+      @urls.add(issue[:url])
+      {
+        id: issue[:cve],
+        name: issue[:advisory_title],
+        level: "MEDIUM",
```

Comment:
Be sure to validate that against the schema.  Also as the various levels are used across multiple scanners you might find opportunity to place them in constants.